components/icons/README.md

```diff
@@ -0,0 +1,100 @@
+# Icons
+
+The `<d2l-icon>` web component can be used in your applications to embed one of the roughly 500 preset SVG icons that make up the Brightspace iconography set. Additionally, it can be used with your own custom SVG icons as well.
+
+## Usage
+
+### Preset Icons
+
+For preset icons, import and use the `<d2l-icon>` web component with the `icon` attribute.
+
+```html
+<script type="module">
+	import '@d2l/brightspace-core/components/icons/icon.js';
```

Comment:
Does this need to match the package name?

Reply:
Haha well we haven't published anything yet, but yes it would. I was assuming this was the point at which we'd start introducing the `@d2l` organization in NPM.

Reply:
Talking with @omsmith, it looks like artifactory has already used the `@d2l` prefix, so using that from NPM is going to cause issues. I'm going to just use the package name here for now. We could still go with something like `@brightspace` or `@brightspaceui` though.